dcos_launch/platforms/aws.py

```diff
@@ -21,14 +21,13 @@
 import time
 
 import pkg_resources
+import retrying
 
 import boto3
-import retrying
+import dcos_launch
```

Comment:
the import ordering should be grouped by stdlib module, 3rd party module, local module https://www.python.org/dev/peps/pep-0008/#imports (second bullet point)